.gitignore

```diff
@@ -42,3 +42,37 @@ docs/warnings.txt
 .DS_Store
 docs/content/examples/*
 docs/modules/*
+examples/00_published/bookpurnong_inversion/bookpurnong_inversion.tar.gz
```

Comment:
Would you mind updating this so we just ignore `examples/00_published/bookpurnong_inversion/*` instead of individually listing all of the files?